tests/backup/backup_orphaned_delete_test.go

```diff
@@ -0,0 +1,362 @@
+				email := fmt.Sprintf("testuser-%v-%s@cnbu.com", role, RandomString(4))
+				err := backup.AddUser(userName, firstName, lastName, email, commonPassword)
+				log.FailOnError(err, "Failed to create user - %s", userName)
+				err = backup.AddRoleToUser(userName, role, fmt.Sprintf("Adding %v role to %s", role, userName))
+				log.FailOnError(err, "Failed to add role for user - %s", userName)
+				if role == backup.ApplicationOwner {
+					appAdminUserName = userName
+				} else if role == backup.InfrastructureOwner {
+					infraAdminUserName = userName
+				} else {
+					appUserName = userName
+				}
+				userNames = append(userNames, userName)
+				userUID, err := backup.FetchIDOfUser(userName)
+				log.FailOnError(err, "Failed to fetch uid for - %s", userName)
+				userIdMap[userName] = userUID
+			}
+		})
+
+		Step(fmt.Sprintf("Adding Credentials from px-admin user and sharing with required users"), func() {
+			log.InfoD(fmt.Sprintf("Adding Credentials from px-admin user and sharing with required users"))
+			adminCtx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-admin ctx")
+			for _, provider := range providers {
+				adminCloudCredUID = uuid.New()
+				adminCredName = fmt.Sprintf("autogenerated-cred-%v", time.Now().Unix())
+				err = CreateCloudCredential(provider, adminCredName, adminCloudCredUID, orgID, adminCtx)
+				log.FailOnError(err, "Failed to create cloud credential - %s", err)
+				log.Infof("Updating cloud credential ownership for non-admin users")
+				if provider != drivers.ProviderNfs {
+					err = UpdateCloudCredentialOwnership(adminCredName, adminCloudCredUID, []string{appAdminUserName}, nil, Read, Invalid, adminCtx, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying updation of ownership for cloudcredential- %s", adminCredName))
+				}
+			}
+		})
+
+		Step(fmt.Sprintf("Adding backup location from px-admin user and sharing with app users"), func() {
+			log.InfoD(fmt.Sprintf("Adding backup location  from px-admin user and sharing with app users"))
+			adminCtx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-admin ctx")
+			for _, provider := range providers {
+				backupLocationName = fmt.Sprintf("autogenerated-backup-location-%v", time.Now().Unix())
+				backupLocationUID = uuid.New()
+				err := CreateBackupLocationWithContext(provider, backupLocationName, backupLocationUID, adminCredName, adminCloudCredUID, getGlobalBucketName(provider), orgID, "", "", adminCtx)
+				log.FailOnError(err, "Failed to add backup location %s using provider %s for px-admin user", backupLocationName, provider)
+				log.Infof("Updating backup location ownership for non-admin users")
+				err = UpdateBackupLocationOwnership(backupLocationName, backupLocationUID, []string{appUserName}, nil, Read, Invalid, adminCtx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying updation of ownership for backupLocation - %s", backupLocationName))
```

Comment:
How about including username in the log message?

dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying ownership update for backup location %s to user %s", backupLocationName, []string{appUserName}))